src/interfaces/account.interface.ts

```diff
@@ -4,11 +4,14 @@ import { LoadableEntityState } from '../store/types';
 import { AccountTokenInterface } from '../token/interfaces/account-token.interface';
 import { ActivityGroup } from './activity.interface';
 
-export interface AccountInterface {
+export interface IAccountBase {
```

Comment:
follow the same  name structure as it is in other places (for ex `AccountInterface`)